src/GitHub.VisualStudio/UI/Views/GitHubPaneViewModel.cs

```diff
@@ -125,6 +128,15 @@ public override void Initialize(IServiceProvider serviceProvider)
                 },
                 true);
 
+           serviceProvider.AddCommandHandler(GuidList.guidGitHubToolbarCmdSet, PkgCmdIDList.helpCommand,
+                () => true,
+                () =>
+                {
+                    browser.OpenUrl(new Uri("https://visualstudio.github.com/docs"));
```

Comment:
There's a const string for this url defined somewhere else. Maybe these urls should go into one of the shared classes we have for defining const data, that way we don't have to hunt down all the instances when we want to change something.